docs/reference/rt-conf-yaml.md

```diff
@@ -0,0 +1,208 @@
+# rt-conf YAML configuration schema
+
+The following schema reflects all settings available in the configuration file of the `rt-conf` tool
+
+There are three top-level dictionaries in the configuration file:, {ref}`kcmd`, {ref}`irqt` and {ref}`cpug`.
+
+(kcmd)=
+## kernel_cmdline
+
+Type: `dict`
+
+_Optional_
+
+{ref}`kernel-boot-parameters` that affects real-time behavior
+
+
+### kernel_cmdline.isolcpus
+
+Type: `string`
+
+_Optional_
+
+A string formatted as {ref}`cpu-lists`. 
+Isolate CPUs from general execution.
+
+### kernel_cmdline.nohz
+
+Type: `enum`
+
+_Optional_
+
+Enable/disable dynamic ticks during idle time.
+
+Valid values are:
+  * `on`: **Enables** dynamic ticks
+  * `off`:**Disables** dynamic ticks
+
+### kernel_cmdline.nohz_full
+
+Type: `string`
+
+_Optional_
+
+A string formatted as {ref}`cpu-lists`. 
+Specifies the adaptive-ticks cpus, which means the specified list of CPUs whose tick will be stopped whenever possible.
+The boot CPU will be forced outside the range to maintain the timekeeping.
+
+### kernel_cmdline.kthread_cpus
+
+Type: `string`
+
+_Optional_
+
+A string formatted as {ref}`cpu-lists`. 
+Specifies the list of CPUs to be allocated for kernel threads.
+
+### kernel_cmdline.irqaffinity
+
+Type: `string`
+
+_Optional_
+
+A string formatted as {ref}`cpu-lists`. 
+Specifies the list of CPUs for IRQ handling.
+
+(irqt)=
+## irq_tuning 
+
+Type: `list[dict]`
+
+_Optional_
+
+A list of configurations including the list of cpus to be applied and the filters.
+Runtime configurations that aren't persisted, related to IRQ affinity tuning.
+
+Example:
+
+```yaml
+irq_tuning:
+  - cpus: "2-3"
+    filter:
+      actions: "iwlwifi"
+      chip_name: "IR-PCI"
+      name: "edge"
+      type: "edge"
+```
+
+### irq_tuning[*].cpus
+
+Type `string`
+
+_Required_
+
+A string formatted as {ref}`cpu-lists`. 
+Specifies the list of CPUs which will handle the matched IRQs on the {ref}`filter <irqfilter>`.
+
+(irqfilter)=
+### irq_tuning[*].filter
+
+Type: `dict`
+
+_Required_
+
+A dictionary with keys related to IRQ properties of `/sys/kernel/irq/<IRQ-num>/`.
+
+#### irq_tuning[*].filter.actions
+
+Type: `regex string`
+
+_Optional_
+
+The IRQ action chain. A comma-separated list of zero or more device names associated with this interrupt.
+For network related, generally is the name of the network interface shown in `ip link show`. 
```

Comment:
I don't find the second sentence very clear. Do you mean to say "For network-related purposes, the IRQ action chain is generally the name of the network interface, shown in `ip link show`."?